src/App.tsx

```diff
@@ -1,27 +1,59 @@
-import React from 'react';
+import React, { useCallback, useState } from 'react';
 import './App.scss';
 import { GoodsList } from './GoodsList';
+import { getAll, get5First, getRedGoods } from './api/goods';
+import { Good } from './types/Good';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+export const App: React.FC = () => {
+  const [goods, setGoods] = useState<Good[]>([]);
+  const setFetchFunction = useCallback(async (
```

Comment:
useCallback here is redundant, it makes sense to use when you lift function as props to memo component, or with useEffect